### ISSUE TITLE
xhtm 1.5.8 does not like newlines between tag attributes

### ISSUE THREAD

**Issue by aral** [dy/xhtm#14]
The following code, with xhtm bound to vhtml:

```js
html`
  <span
    id='status'
  >OK</span>
`
```

Results in:

```html
<spanid='status'>OK</spanid='status'>
```

Instead of:

```html
<span id='status'>OK</span>
```

**Comment by dy** [comment]
Check v1.5.9

**Comment by dy** [comment]
v.1.5.10 is even better.

**Comment by aral** [comment]
Thank you; it issue introduced in 1.5.8 was fixed for me in 1.5.9 :) Trying 1.5.10 now :)
